src/components/videoPage/VideoAIMetadataContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import s from 'src/components/videoPage/style.module.less';
+import Badge from '@boclips-ui/badge';
+import { Typography } from '@boclips-ui/typography';
+import React from 'react';
+
+interface Props {
+  isLoading: boolean;
+  metadata?: string[];
+  type: string;
+}
+export const VideoAIMetadataContent = ({
+  isLoading,
+  metadata,
+  type,
+}: Props) => {
+  const getAIBadge = () =>
+    isLoading ? (
+      <LoadingOutlined className={s.spinner} />
+    ) : (
+      <Badge value="AI generated" />
+    );
+
+  const showContent = () => (
+    <ul className={s.outcomeList}>
+      {metadata.map((elem: string) => (
+        <Typography.Body as="li" size="small" className="text-gray-800">
```

Comment:
We need key for listing items, more info about it can be found here: https://legacy.reactjs.org/docs/lists-and-keys.html

Reply:
or here: https://react.dev/learn/rendering-lists